tests/expressions/tests.py

```diff
@@ -174,6 +174,19 @@ def test_update_with_fk(self):
             ordered=False
         )
 
+    def test_update_with_none(self):
+        Number.objects.create(integer=1, float=1.0)
+        Number.objects.create(integer=2)
+        self.assertEqual(
+            Number.objects.filter(float__isnull=False).update(float=Value(None)),
+            1)
```

Comment:
It may be useful to remind the reader here that `update()` returns the number of rows matched, not the number updated.

(actually, it may make better sense to just drop the assertion. The next assert really checks everything).